webapp/src/components/sidebar_buttons/sidebar_buttons.tsx

```diff
@@ -1,36 +1,45 @@
-import React from 'react';
-import {Tooltip, OverlayTrigger} from 'react-bootstrap';
+import React, { PureComponent, ReactElement } from 'react';
+import { Tooltip, OverlayTrigger } from 'react-bootstrap';
 import PropTypes from 'prop-types';
-import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
-
-import {RHSStates, connectUsingBrowserMessage} from 'src/constants';
-import {isDesktopApp} from 'src/utils/user_agent';
+import { makeStyleFromTheme, changeOpacity } from 'mattermost-redux/utils/theme_utils';
+
+import { RHSStates, connectUsingBrowserMessage } from 'src/constants';
+import { isDesktopApp } from 'src/utils/user_agent';
+
+import { GitLabIssuesIcon, GitLabMergeRequestIcon, GitLabReviewsIcon, GitLabTodosIcon } from './button_icons';
+import { Placement } from 'react-bootstrap/esm/Overlay';
+
+interface SidebarButtonsProps {
+    theme: any;
+    connected: boolean;
+    username: string;
+    org: string;
+    clientId: string;
+    gitlabURL: string;
+    reviews: any[];
```

Comment:
One thing we want to avoid is putting `any` on the types, as this can sort of defeat the purpose of using typescript. We're only checking the length of the arrays in this file, so we don't necessarily need the type. If you would like to type them from these types you can. Otherwise we can just use `unknown` instead of `any`

```suggestion
    reviews: unknown[];
```